src/Data/Product/Relation/Binary/Lex/Strict.agda

```diff
@@ -179,13 +182,11 @@ module _ {_≈₁_ : Rel A ℓ₁} {_<₁_ : Rel A ℓ₂}
          (x₁≈y₁ , x₂≈y₂)
          [ x₁≯y₁ , x₂≯y₂ ∘ proj₂ ]
 
-module _ {_<₁_ : Rel A ℓ₁} {_<₂_ : Rel B ℓ₂} where
-
-  -- Currently only proven for propositional equality
-  -- (unsure how to satisfy the termination checker for arbitrary equalities)
+module _ {_≈₁_ : Rel A ℓ₁} {_<₁_ : Rel A ℓ₂} {_<₂_ : Rel B ℓ₃} where
```

Comment:
Unfortunately adding `_≈₁_` to the telescope here parameterises the original `×-wellFounded` proof by `_≈₁_` as well. Of course it doesn't use `_≈₁_` so users will get unsolved metas.

I would add `_≈₁_` as an argument directly to the type of `×-wellFounded'`. And maybe we could call it just `_≈_` as there's no second equality relation?

Reply:
I use `_≈₁_` in the private declaration of `_<ₗₑₓ'_`  so making it directly an argument wouldn't work , so maybe putting the declaration in an other module ?

Also i named it like that because it's the name used in the definition of `×-Lex`

Reply:
Okay yes, then a separate module is needed.